rapids-4-spark-tools/README.md

```diff
@@ -1,78 +1,197 @@
 # Spark Qualification and Profiling tools
 
-The qualification tool is used to rank a set of applications to determine if the RAPIDS Accelerator for Apache Spark is a good fit for those applications.
+The qualification tool is used to rank a set of applications to determine if the RAPIDS Accelerator for Apache Spark 
+is a good fit for those applications. Works with CPU generated event logs.
 
 The profiling tool generates information which can be used for ranking applications debugging and profiling.
-Information such as Spark version, executor information, Properties and so on.  Works with both cpu and gpu generated event logs.
+Information such as Spark version, executor information, properties and so on.  
+Works with both CPU and GPU generated event logs.
 
 (The code is based on Apache Spark 3.1.1 source code, and tested using Spark 3.0.x and 3.1.1 event logs)
 
 ## Prerequisites
-1. Require Spark 3.1.1 or newer installed
+- Spark 3.1.1 or newer installed
```

Comment:
we can leave this for now, I want to make it run with < 3.1.1, working on that now but can update with that pr

Reply:
Sure, once the PR for that is submitted, you can include some modification for README